toloka_provider/sensors/toloka.py

```diff
@@ -48,3 +44,45 @@ def poke(self, context: 'Context') -> bool:
         self.log.info(f'Pool {pool_id} - {percentage}%')
 
         return pool.is_closed()
+
+
+class WaitAppBatchSensor(BaseSensorOperator):
+    """
+    Wait given App batch until it enters an of poke_on_statuses status.
+
+    :param app_project: Either an `AppProject` object or it's config or an app_project_id value.
+    :param app_project: Either an `AppBatch` object or it's config or a batch_id value.
+    :param toloka_conn_id: Airflow connection with toloka credentials.
+    :param poke_on_statuses: Container of AppBatch.Status
+    """
+
+    template_fields: Sequence[str] = ('app_project', 'batch', 'poke_on_statuses')
```

Comment:
I think toloka_conn_id should be templated too, because we want to get toloka_conn_id from dag params sometimes. And add it in the WaitPoolSensor too, pls